### PR TITLE
Ensure install test are executed

### PR DESCRIPTION
**Pull request by jlojosnegros**
Some test were not executed and fail due to some timing issues.
Create a new github action to execute them at least as frecuently as e2e
tests

### CODE REVIEW

File: test/e2e/install/install_test.go [hunk 1]
```diff
@@ -201,9 +201,14 @@ var _ = Describe("[Install] durability", func() {
...
+			By("waiting for the DaemonSet to be created..")
 			uid := nroObj.GetUID()
-			ds, err := getDaemonSetByOwnerReference(uid)
-			Expect(err).ToNot(HaveOccurred())
+			ds := &v1.DaemonSet{}
+			Eventually(func() error {
+				var err error
+				ds, err = getDaemonSetByOwnerReference(uid)
+				return err
+			}, 5*time.Minute, 10*time.Second).Should(BeNil())
```
Comment (fromanirh) [inline test/e2e/install/install_test.go:211]: this is a good change we should merge anyway. Better to split on a separate PR we can merge quickly

Comment (jlojosnegros) [reply test/e2e/install/install_test.go:211]: The problem I see with splitting this in another PR is that if this test is not being executed we cannot be sure it is really ok

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -87,3 +87,76 @@ jobs:
...
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
+
+    - name: Verify modules
+      run: go mod verify
+
+    - name: build test binary
+      run: |
+        make binary-e2e-rte binary-e2e-install binary-e2e-uninstall
```
Comment (fromanirh) [inline .github/workflows/e2e.yml:113]: The problem is our github actions infra is already overtaxed, and I'm not convinced here that the benefits are greater than the costs. Let's keep the k8s (through kind) coverage minimal (but sufficient) until there is a crystal clear case to enhance it. Our main focus is OCP anyway.

File: test/e2e/install/install_test.go (hunk 1 above)
Comment (Tal-or) [reply test/e2e/install/install_test.go:211]: The test is executed under the ```ci-e2e-install``` lane.
Why do you think it's not?

File: test/e2e/install/install_test.go [hunk 3]
```diff
@@ -201,9 +201,14 @@ var _ = Describe("[Install] durability", func() {
...
 			Expect(err).ToNot(HaveOccurred())
 
+			By("waiting for the DaemonSet to be created..")
 			uid := nroObj.GetUID()
-			ds, err := getDaemonSetByOwnerReference(uid)
-			Expect(err).ToNot(HaveOccurred())
+			ds := &v1.DaemonSet{}
+			Eventually(func() error {
+				var err error
+				ds, err = getDaemonSetByOwnerReference(uid)
```
Comment (Tal-or) [inline test/e2e/install/install_test.go:209]: I would add a ```klog.Warningf``` here which says something like: "ds is not ready/found yet"

**Comment by jlojosnegros** [comment]
No need to change GH actions.
Code change already covered by #132
